packages/sdk/src/cosmwasmclient.ts

```diff
@@ -159,11 +175,16 @@ export class CosmWasmClient {
       }
     } else if (isSearchBySentFromOrToQuery(query)) {
       // We cannot get both in one request (see https://github.com/cosmos/gaia/issues/75)
-      const sent = await this.txsQuery(withFilters(`message.sender=${query.sentFromOrTo}`));
-      const received = await this.txsQuery(withFilters(`transfer.recipient=${query.sentFromOrTo}`));
+      const sentQuery = withFilters(`message.module=bank&message.sender=${query.sentFromOrTo}`);
+      const receivedQuery = withFilters(`transfer.recipient=${query.sentFromOrTo}`);
```

Comment:
Very good queries. I saw the comment above, and this is exactly what I would have suggested.

But then, I wonder, do we want to capture just `MsgSend`, or all events that move funds (like when a contract sends tokens, or if you send to a contract)? `receivedQuery` works in the second case (got from contract), but `sentQuery` only works on the "plain" sends. This should be somehow symetric.|

Suggestion:
* Either add `message.module=bank&` to the received query
* Or just use `transfer.sender` on the first query (which only works on our patched sdk, but I think this is fine). 

You can always make two sentQueries - the current one which always matches on basic ones even on upstream:cosmos-sdk, and the "enhanced" `transfer.sender` one, and then de-duplicate them as is already done (to not double flag a self-send)

Reply:
I think for simplicity we should stick with bank transfers and follow approach number 1. Otherwise 🤯 when trying to cover everything with a simple "sentFromOrTo".